### PR TITLE
Update create-a-dlp-policy-from-a-template.md

### PR DESCRIPTION
**Pull request by msbemba**
updated Exchange email accounts as the options show only distribution groups

### CODE REVIEW

**Comment by msbemba** [comment]
Issue# https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2049

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/create-a-dlp-policy-from-a-template.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ In this example, you'll create a DLP policy that identifies U.S. PII data, which
     
   - Choose **Let me choose specific locations** \> **Next**. For this example, choose this.
     
-    To include or exclude an entire location such as all Exchange email or all OneDrive accounts, switch the **Status** of that location on or off. 
+    To include or exclude only specific Exchange email accounts  switch the **Status** of that location on to choose specific distribution group. Based on selection, policy will target or exclude the members of the group.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-a-dlp-policy-from-a-template.md]:
```suggestion
    To include or exclude only specific Exchange email accounts, switch the **Status** of that location on to choose a specific distribution group. Based on the selection, the policy will target or exclude the members of the group.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mypil** [comment]
@msbemba - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@chrfox - PR has been copy edited and is ready for final review. This is based on issue #2049

Thank you.

**Comment by chrfox** [comment]
suggested wording is confusing, the UI makes the functionality clear

**Pull request closed without merging**